docfx.json

```diff
@@ -101,8 +101,8 @@
     "globalMetadataFiles": [],
     "fileMetadataFiles": [],
     "template": [
-      "nano1",
-      "templates/nano1"
+      "default",
```

Comment:
won't these names have to match?

Reply:
I think not https://dotnet.github.io/docfx/docs/template.html#create-a-custom-template

And just because of this, the last assembly did not pass.

Reply:
OK! From the docs, that seems to be the case. 
Wanted to check if it wasn't a typo or something.